js/whitebit.js

```diff
@@ -462,23 +463,23 @@ module.exports = class whitebit extends Exchange {
         //           {...}
         //      }
         //
-        const currenciesIds = Object.keys (response);
-        const withdrawFees = {};
-        const depositFees = {};
-        for (let i = 0; i < currenciesIds.length; i++) {
-            const currency = currenciesIds[i];
+        const result = {};
+        if (codes === undefined) {
+            codes = Object.keys (response);
+        }
+        for (let i = 0; i < codes.length; i++) {
```

Comment:
@AlgoQ ok so whitebit is returning some coins alongside the network
![image](https://user-images.githubusercontent.com/43336371/199474779-6ed52e3a-ac60-4bf7-bc80-1bdf4227e59b.png)

So for instance currently `fetchTransactionFees(['USDT'])` won't work.

I would suggest doing similar to what was done with bitstamp (group all related fields)